Dockerfile

```diff
@@ -8,11 +8,16 @@
 #
 FROM mikefarah/yq as builder
 RUN apk add --no-cache bash
-COPY .htaccess README.md /build/
+COPY .htaccess README.md check_plugins_location.sh check_plugins_images.sh check_plugins_viewer_mandatory_fields.sh index.sh index_v2.sh index_util.sh set_plugin_dates.sh util.sh /build/
```

Comment:
I wonder if we can copy all `sh` files without listing their names.

Reply:
done